client/src/app/app.component.html

```diff
@@ -43,6 +43,7 @@ <h1>ODS Provisioning</h1>
   <div class="container">
     <footer>
       © 2020 <a href="https://opendevstack.org" target="_blank">opendevstack.org</a>
+      <div class="trademark">Atlassian Jira, Confluence and Bitbucket logos are registered logomarks of Atlassian. Redhat OpenShift logo is a registered logomark of Red Hat, Inc.</div>
```

Comment:
Not sure about legal requirements for using commercial logos in an open source app ... I'd prefer to better place this hint in the footer than not. What's your opinion @michaelsauter @clemensutschig ?